Formula/uncrustify.rb

```diff
@@ -16,8 +16,13 @@ class Uncrustify < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on :macos
```

Comment:
Seeing that there is a linux bottle, it worked before the patch.

Reply:
Possibly, but there's no Linux CI on the linked PR?

Reply:
Linux CI arrived later, as a part of the merging of homebrew-core and linuxbrew-core.

Reply:
Yes, that much was clear. I can't help but lament the shifting goal posts; makes it significantly harder to contribute to homebrew.

Reply:
Let's get rid of this line so we can see what the Linux failure is and try to fix it.

Reply:
https://github.com/Homebrew/homebrew-core/pull/86327/checks?check_run_id=3781104013#step:7:279 

Looks like some bad compiler setup